exercises/two-fer/README.md

```diff
@@ -14,24 +14,5 @@ If no name is given, the result should be "One for you, one for me."
 For installation and learning resources, refer to the
 [exercism help page](http://exercism.io/languages/ruby).
 
-For running the tests provided, you will need the Minitest gem. Open a
```

Comment:
I think this section on how to run the tests is required, it was just the "For example.." line that I thought needed to be removed.

Sorry if my earlier request was confusing.

Reply:
I solved this problem once and for all by autogenerating READMEs: https://github.com/exercism/ruby/pull/721

Reply:
Ok, the testing instructions will need to be part of the README in this PR though.